modern/src/LoginPage.js

```diff
@@ -87,54 +88,78 @@ const LoginPage = () => {
-        <form onSubmit={handleLogin}>
-
-          <TextField
-            margin='normal'
-            required
-            fullWidth
-            error={failed}
-            label={t('userEmail')}
-            name='email'
-            value={email}
-            autoComplete='email'
-            autoFocus
-            onChange={handleEmailChange}
-            helperText={failed && 'Invalid username or password'} />
-
-          <TextField
-            margin='normal'
-            required
-            fullWidth
-            error={failed}
-            label={t('userPassword')}
-            name='password'
-            value={password}
-            type='password'
-            autoComplete='current-password'
-            onChange={handlePasswordChange} />
-
-          <FormControl fullWidth margin='normal'>
-            <div className={classes.buttons}>
-              <Button type='button' variant='contained' onClick={handleRegister} disabled={!registrationEnabled}>
-                {t('loginRegister')}
-              </Button>
-              <Button type='submit' variant='contained' color='primary' disabled={!email || !password}>
+        <form className={classes.form} onSubmit={handleLogin}>
+          <Grid container direction='column' spacing={3}>
+            <Grid item>
+              {matches && <Logo fill='#333366'/>}
+            </Grid>
+            <Grid item>
+              <TextField
+                required
+                fullWidth
+                error={failed}
+                label={t('userEmail')}
+                name='email'
+                value={email}
+                autoComplete='email'
+                autoFocus
+                onChange={handleEmailChange}
+                helperText={failed && 'Invalid username or password'}
+                variant='filled' />
+            </Grid>
+            <Grid item>
+              <TextField
+                required
+                fullWidth
+                error={failed}
+                label={t('userPassword')}
+                name='password'
+                value={password}
+                type='password'
+                autoComplete='current-password'
+                onChange={handlePasswordChange}
+                variant='filled' />
+            </Grid>
+            <Grid item>
+              <Button 
+                type='submit'
+                variant='contained' 
+                color='secondary'
+                disabled={!email || !password}
+                fullWidth>
                 {t('loginLogin')}
               </Button>
-            </div>
-          </FormControl>
+            </Grid>
+            <Grid item container>
+              <Grid item>
+                <Button onClick={handleRegister} disabled={!registrationEnabled} color="secondary">
+                  {t('loginRegister')}
+                </Button>
+              </Grid>
+              <Grid item xs>
+                <FormControl variant="filled" fullWidth>
```

Comment:
@tananaev Looks like he'll need some assistance to implement the language switcher

Reply:
We can probably keep it as is and implement in a separate PR.

Reply:
That works as well